state/indexer/sink/psql/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TYPE block_event_type AS ENUM ('begin_block', 'end_block', '');
+CREATE TABLE block_events (
```

Comment:
Can we also add a `created_at TIMESTAMPTZ NOT NULL` to each table?

Reply:
Sure.